library/ZendService/Twitter/Response.php

```diff
@@ -152,4 +160,52 @@ public function toValue()
     {
         return $this->jsonBody;
     }
+
+    /**
+     * Retun the array of response rate info
+     *
+     * @return array
+     */
+    public function getRateLimit()
```

Comment:
Hi! :)

Why is it better than an array?

The method is needed as the objects using the response need to be able to have access to this data. 

Cheers!
=C=